docs/Server.md

```diff
@@ -961,6 +961,26 @@ fastify.ready(() => {
 })
 ```
 
+<a name="print-plugins"></a>
+#### printPlugins
+
+`fastify.printPlugins()`: Prints the representation of the internal plugin tree used by the avvio, useful for debugging require order issues.<br/>
+*Remember to call it inside or after a `ready` call.*
+
+```js
+fastify.get('/test', () => {})
+fastify.get('/test/hello', () => {})
+fastify.get('/hello/world', () => {})
+
+fastify.ready(() => {
+  console.log(fastify.printPlugins())
```

Comment:
🤔 should this be a `fastify.log.info`? By using `console.log` we are promoting adding what many would consider a bug to production code.

Reply:
This should be console.log: the scope is printing to the terminal

Reply:
That may be, but it could still interfere with shipping the JSON logs. I think there should at least be a comment in this example that the `console.log` should be disabled prior to running the code in production.

A better fix would be for us to add some sort of "console logger" for things like this that:

1. Is enabled via a server option
2. writes to `stderr` or another fd other than `stdout`

Reply:
We can use `console.error` instead.